src/api/Tasks.js

```diff
@@ -48,10 +48,18 @@ class Tasks extends Base {
             return;
         }
 
-        // Manually create the entries array if creating/updating a task
-        const entries = data.entries ? data.entries : [data];
+        // We don't have entries when updating/creating a task
+        if (!data.entries) {
+            const task = {
```

Comment:
See above comment on extracting as util method you can reuse here and in the `map`